src/commands/scanner/run.ts

```diff
@@ -222,46 +222,71 @@ export default class Run extends ScannerCommand {
 	}
 
 	private processOutput(rrr: RecombinedRuleResults): AnyJson {
```

Comment:
Feels like run.ts has been growing a lot over time - we should consider moving out methods to specialist classes. For example, all the output-process-related methods could probably live in their own class under `lib`.

Reply:
I agree with doing that in the long-term. We should log a work record for it.